feat: adding recipes tracker workshop (#56456)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5633.md

---
id: 66fbcf750a62784cf11f5633
title: Step 9
challengeType: 1
dashedName: step-9
---

# --description--

> Create a `getDifficultyLevel` function with `cookingTime` as the parameter. 

Reviewer: ```suggestion
Create a `getDifficultyLevel` function that takes `cookingTime` as a parameter.
```

> Create `getDifficultyLevel` function with `cookingTime` as the parameter. Inside the function. Create the conditional to help you assign an `easy`, `medium`, and `hard` difficulty levels based on the `cookingTime` of each recipe.

Reviewer 2: It would be better to remove this second sentence. we want campers to practice their problem solving and it should be up to them to think through what they need to do here. Plus they will have had some practice with conditions at this point. 

```suggestion
Create a `getDifficultyLevel` function with `cookingTime` as the parameter. 
```
